datadog_checks_dev/datadog_checks/dev/tooling/commands/validate/licenses.py

```diff
@@ -160,29 +254,98 @@ async def get_data(url):
                 info['name'],
                 info['author'] or info['maintainer'] or info['author_email'] or info['maintainer_email'] or '',
                 info['license'],
+                info['home_page'],
                 {extract_classifier_value(c) for c in info['classifiers'] if c.startswith('License ::')},
             )
 
 
 async def scrape_license_data(urls):
-    package_data = defaultdict(lambda: {'copyright': set(), 'licenses': [], 'classifiers': set()})
+    package_data = defaultdict(
+        lambda: {'copyright': dict(), 'licenses': set(), 'classifiers': set(), 'home_page': None, 'author': None}
+    )
 
     async with Pool() as pool:
-        async for package_name, package_copyright, package_license, license_classifiers in pool.map(get_data, urls):
+        async for package_name, package_author, package_license, home_page, license_classifiers in pool.map(
+            get_data, urls
+        ):
             data = package_data[package_name]
-            if package_copyright:
-                data['copyright'].add(package_copyright)
+            if package_author:
+                data['author'] = package_author
 
             data['classifiers'].update(license_classifiers)
             if package_license:
                 if ' :: ' in package_license:
                     data['classifiers'].add(extract_classifier_value(package_license))
                 else:
-                    data['licenses'].append(package_license)
+                    data['licenses'].add(package_license)
+
+            if home_page:
+                data['home_page'] = home_page
 
     return package_data
 
 
+def update_copyrights(package_name, license_id, data, ctx):
+    home_page = data['home_page']
+    repo_url = PACKAGE_REPO_OVERRIDES.get(package_name) if PACKAGE_REPO_OVERRIDES.get(package_name) else home_page
+    if repo_url:
+        cp, created_date = scrape_copyright_data(repo_url, ctx)
+        if cp:
+            data['copyright'][license_id] = cp
+    if data['author'] and not data['copyright'].get(license_id):
+        cp = 'Copyright {}{}'.format(created_date, data['author'])
```

Comment:
`created_date` might be undefined at this point I think

Reply:
good catch! thanks